### PR TITLE
Update README.ubuntu.bash for Python 3

### PR DESCRIPTION
**Pull request by adam8157**
Greenplum is using Python 3 now, update the script.

If you are using other platforms, please help updating the corresponding script and python dependencies files.

### CODE REVIEW

File: README.ubuntu.bash [hunk 1]
```diff
@@ -32,13 +32,11 @@ apt-get install -y \
...
+  python3-pip \
+  python3-psutil \
+  python3-pygresql \
+  python3-yaml \
   zlib1g-dev
 
-
-pip install conan
-
+pip3 install conan
```
Comment (asimrp) [inline README.ubuntu.bash:42]: Is conan still needed?

**Comment by ashwinstar** [comment]
@edespino fyi, we talked about the README's offlist. Thanks Adam.

File: README.ubuntu.bash (hunk 1 above)
Comment (adam8157) [reply README.ubuntu.bash:42]: Not sure, ORCA needed it before, I'm gonna leave it to someone else.😂

**Review by asimrp — APPROVED** [review]

**Comment by adam8157** [comment]
@ashwinstar I would suggest we shrink the contents, just publish the essential platforms and their guides, then it would be much easier to keep updated.

README.*.md, python-dependencies.txt, and python-developer-dependencies.txt have some contents not necessary IMO.

**Pull request merged**